examples/gno.land/r/gnoland/faucet/faucet.gno

```diff
@@ -69,7 +69,7 @@ func Render(path string) string {
 	output += ufmt.Sprintf("Admin: %s\n\n ", gAdminAddr.String())
 	output += ufmt.Sprintf("Controllers:\n\n ")
 
-	for i := 0; i < gControllersSize; i++ {
+	for i := 0; i < gControllers.Size(); i++ {
```

Comment:
Good find :+1: 

Maybe something else we could add is to rename the `gControllersSize` to `gControllersMaxSize`, so making it clear that it is not necessarily always 10?

Reply:
good point 
c3186cc